src/services/api/api.js

```diff
@@ -0,0 +1,171 @@
+import * as axios from 'axios';
+import jwtDecode from 'jwt-decode';
+import { createUser, editUser, getAllUsers, getUserData, removeUserData } from '../users-services ';
+import { createTask, getAllTasks, getTaskData } from '../tasks-services';
+import { logout, singInEmailAndPassword } from '../auth-services';
+
+function getToken() {
+  return localStorage.getItem('token');
+}
+
+const instance = axios.create({
+  baseURL: 'https://dims-core-api.herokuapp.com/api/',
+  headers: {
+    accept: 'application/json',
+    'Content-Type': 'application/json',
```

Comment:
As far as I remember these headers are included by default. You can omit them.

Reply:
Removed. Thanks.